services_app/src/main/java/org/opendatakit/services/sync/actions/fragments/SyncFragment.java

```diff
@@ -729,8 +737,15 @@ private void performActionOnSyncComplete(SyncStatus syncStatus){
   }
 
   public void showAlertDialog(String title, String message, DialogInterface.OnClickListener onPositiveButtonClick){
-    androidx.appcompat.app.AlertDialog alertDialog = new androidx.appcompat.app.AlertDialog
-            .Builder(requireActivity())
+
+    /**
+     * New dialog styling
+     * MaterialAlertDialogBuilder is standard for all ODK-X Apps
+     * OdkAlertDialogStyle present in AndroidLibrary is used to style this dialog
+     * @params change MaterialAlertDialogBuilder to AlertDialog.Builder in case of any error and remove R.style... param!
+     */
+
+    androidx.appcompat.app.AlertDialog alertDialog = new MaterialAlertDialogBuilder(requireActivity(),R.style.OdkXAlertDialogStyle)
```

Comment:
why isn't the type "AlertDialog"? Why the fully qualified " androidx.appcompat.app.AlertDialog"

Reply:
I just fixed it now